### ISSUE TITLE
Uncaught TypeError: Cannot destructure property `prefix` of 'undefined' or 'null'.

### ISSUE THREAD

**Issue by normcyr** [atom/atom#20533]
Steps to reproduce:

1. Launch atom program.
2. Red error window appears on the right.

**Atom**: 1.45.0 x64
**Electron**: 4.2.7
**OS**: Debian GNU/Linux (stretch)
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: Cannot destructure property `prefix` of 'undefined' or 'null'.

```
At /usr/share/atom/resources/app/static/<embedded>:11

TypeError: Cannot destructure property `prefix` of 'undefined' or 'null'.
    at Object.add (/usr/share/atom/resources/app/static/<embedded>:11:232701)
    at atom.config.transact (/usr/share/atom/resources/app/static/<embedded>:11:229087)
    at Config.transact (/usr/share/atom/resources/app/static/<embedded>:11:331514)
    at loadUserSnippets.o (/usr/share/atom/resources/app/static/<embedded>:11:229015)
    at n.isFile.loadSnippetsFile.o (/usr/share/atom/resources/app/static/<embedded>:11:229397)
    at p.readFile (/usr/share/atom/resources/app/static/<embedded>:11:232594)
    at /usr/share/atom/resources/app/static/<embedded>:11:581672
    at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
```

### Commands

```
```

### Non-Core Packages

```
atom-black-syntax 0.33.0 
atom-black-ui 0.26.0 
atom-gitbook 0.1.4 
atom-material-syntax 1.0.8 
autocomplete-bibtex 1.2.7 
bson-converter 0.5.0 
busy-signal 2.0.1 
chester-atom-syntax 0.3.2 
framer-syntax 1.1.0 
framer-ui 2.0.0 
gl-dark-syntax 3.0.0 
gtk-dark-theme 0.1.4 
Hydrogen 2.14.1 
hydrogen-launcher 1.2.2 
intentions 1.1.5 
language-latex 1.2.0 
language-markdown 0.37.0 
language-pfm 0.82.0 
linter 2.3.1 
linter-pyflakes 0.3.2 
linter-ui-default 1.8.1 
markdown-pdf 2.3.3 
markdown-preview-plus 3.11.5 
mathjax-wrapper 0.1.0 
minimal-syntax 0.22.0 
minimal-syntax-dark 0.12.4 
nord-atom-syntax 0.10.0 
nord-atom-ui 0.12.0 
one-o-eight-syntax 0.39.0 
pubster-syntax 3.1.0 
python-black 0.7.0 
seti-ui 1.11.0 
wordcount 3.1.0 
```

**Comment by lkashef** [comment]
Hello @normcyr, Thanks for the report.

TL;DR Please run Atom in Safe Mode

There were debugging steps that we provide on the issue template that help you write a better report and do basic debugging so we can efficiently help you, one of these essential steps is to run atom in safe mode which will disable all the non-core packages to make sure the reported bug is related to atom core.

As we treat issues as our backlog I will go ahead and close this one, if the problem persists feel free to create a new issue and make sure you follow the `Prerequisites` section.

**Comment by LamLamLLC** [comment]
Sorry
